server/@types/prisonRegisterApi/index.d.ts

```diff
@@ -48,6 +127,80 @@ export interface components {
       developerMessage?: string
       moreInfo?: string
     }
+    /** List of address for this prison */
+    AddressDto: {
+      /** Unique ID of the address */
+      id: number
+      /** Address line 1 */
+      addressLine1?: string
+      /** Address line 2 */
+      addressLine2?: string
+      /** Village/Town/City */
+      town: string
+      /** County */
+      county?: string
+      /** Postcode */
+      postcode: string
+      /** Country */
+      country: string
+    }
+    /** Prison Information */
+    PrisonDto: {
```

Comment:
These are the types we are _really_ interested in - `PrisonDto` and `AddressDto`